openslides_backend/action/create_action_with_inferred_meeting.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Dict, Type
+
+from ..models.fields import BaseGenericRelationField, BaseRelationField
+from ..shared.exceptions import ActionException
+from ..shared.patterns import FullQualifiedId
+from .generics import CreateAction
+
+
+class CreateActionWithInferredMeetingMixin(CreateAction):
+    """
+    Mixin to autmatically set the meeting_id on create if it's not given in the payload.
+    If related_collection_for_meeting is not given, the relation_field_for_meeting must be
+    a generic field and therefore be a fqid.
+    """
+
+    relation_field_for_meeting: str
+
+    def update_instance_with_meeting_id(
+        self, instance: Dict[str, Any]
+    ) -> Dict[str, Any]:
+        field = self.model.get_field(self.relation_field_for_meeting)
+        assert isinstance(field, BaseRelationField)
+        id = instance[self.relation_field_for_meeting]
+        if isinstance(field, BaseGenericRelationField):
+            fqid = id
+        else:
+            assert not isinstance(field.to, list)  # for mypy
+            fqid = FullQualifiedId(field.to, id)
+        # Fetch meeting_id
+        related_model = self.fetch_model(fqid, ["meeting_id"])
+        if not related_model.get("meeting_id"):
+            raise ActionException(
+                f"Referenced model in field {self.relation_field_for_meeting} has no meeting id."
+            )
+        instance["meeting_id"] = related_model["meeting_id"]
+        return instance
+
+
+class CreateActionWithInferredMeeting(
+    CreateActionWithInferredMeetingMixin, CreateAction
+):
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        return self.update_instance_with_meeting_id(instance)
+
+
+def get_create_action_with_inferred_meeting(
+    relation_field_for_meeting: str,
+) -> Type[CreateActionWithInferredMeeting]:
+    return type(
```

Comment:
Please add a docstring here that this is a shortcut to setup a create action class with this new mixin and the given field name. Somethink like this:
```
"""
Shortcut for a create action with inferred meeting.

    MyAction = get_create_action_with_inferred_meeting("foo")

equals to

    class MyAction(CreateActionWithInferredMeeting):
        relation_field_for_meeting = "foo"
"""
```